srcpkgs/backintime-qt/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'backintime-qt'
+pkgname=backintime-qt
+version=1.2.0
+revision=1
+archs=noarch
+wrksrc=backintime-${version}
+build_wrksrc="qt"
+build_style=configure
+configure_args="--python3"
+pycompile_dirs="/usr/share/backintime"
+hostmakedepends="python3"
+depends="python3 rsync python3-dbus python3-PyQt5 python3-PyQt5-dbus libnotify xdpyinfo backintime"
+short_desc="Qt Gui for backintime"
+maintainer="Juan RP <xtraeme@voidlinux.eu>"
+license="GPL-2"
```

Comment:
Find out whether it is GPL-2.0 only or GPL-2.0-or-later, and update. Same for backintime.